### PR TITLE
Allow Custom Component for popup

### PR DESCRIPTION
**Pull request by Axosoft-Ashley**

### CODE REVIEW

**Comment by Axosoft-Ashley** [comment]
Dont merge this yet... something is wrong here its not my original commit... idk what happened to this PR

**Comment by Axosoft-Ashley** [comment]
Once this goes in i can point the calendar pr to @master for us

File: src/Month.js [hunk 1]
```diff
@@ -264,6 +264,8 @@ class MonthView extends React.Component {
     let overlay = (this.state && this.state.overlay) || {}
     let { components } = this.props
 
+    let PopupComponent = components.popup || Popup
```
Comment (axosoft-dans) [inline src/Month.js]: should be `const` ?

Comment (Axosoft-Ashley) [reply src/Month.js]: copied how the two props above this one was being used, but i confirmed they do use const so ill change it.

**Review by jdgarcia — APPROVED** [review]

**Pull request merged**